plugins/storage/fs/test/unit/test_fs.py

```diff
@@ -37,6 +38,7 @@ async def save_and_load_file():
     path = await fs.store(key, test_fs)
     assert path == f'/tmp/{key}/{key}.json'
     loaded = await fs.get(key, datatype=FsMockData)
+    await fs.delete(key)
```

Comment:
This does nothing? It should be checked at least if the remove method in the filesystem is called.

I would add a test with multiple keys to delete to check the for loops is working.